Utils/alerts.py

```diff
@@ -52,7 +61,87 @@ def convert_dataframe_to_bytes(dataframe):
         RawMessage={"Data": message.as_string()},
     )
 
-    print(response)
-
+    logger.info(response)
+
+
+class SlackMessaging:
+    SLACK_TOKEN = os.getenv('SLACK_APP_TOKEN')
+    SLACK_CHANNEL = os.getenv('SLACK_CHANNEL')
+
+    def __init__(self):
+        self.client = WebClient(token=SlackMessaging.SLACK_TOKEN)
+
+    # ========================================
+    # Message related methods
+    # ========================================
+    def validate_slack_token(self):
+        try:
+            response = self.client.auth_test()
+            if response["ok"]:
+                logger.debug("Token is valid.")
+                return True
+        except SlackApiError as e:
+            logger.error(f"Error: Unable to validate Slack Token: {e.response['error']}")
+            # Send email notification:
+            ses_client = boto3.client("ses", region_name="us-east-1")
+            message = MIMEMultipart()
+            message["Subject"] = "Slack Error: API Token Invalid"
+            body = MIMEText(f"Slack Error: {e.response['error']}.")
+            message.attach(body)
+            response = ses_client.send_raw_email(
+                Source="hive@u4u-email.com",
```

Comment:
This is outdated... the u4u-email.com domain doesn't exist anymore. I was able to get us to be able to send an email from an [at]unrefugees[dot]org email so they now originate from the hive[at]unrefugees[dot]org.

See here for more details https://github.com/USAFORUNHCRhive/Integrations/blob/main/src/Salesforce/SESEmailer/emailer.py